### DISCUSSION TITLE
Drop down bug

### DISCUSSION THREAD

**Discussion by LeSe0** [heroui-inc/heroui#696] [Help]
Hey guys, please help me. How to fix bug when dropdown appear from left size, disable animation not help me. It's work like this only when i reload page and click on dropdown button first time

**Comment by jrgarciadev** [comment]
Hey @LeSe0 could you share a codesandbox/repository?

**Reply by LeSe0** [reply]
I can do it tomorrow, It's ok?

**Reply by jrgarciadev** [reply]
for sure 👍🏻

**Reply by LeSe0** [reply]
Hello, it's a test repo with bug
https://github.com/LeSe0/NextUI

**Reply by LeSe0** [reply]
When you run project, you can see that when you open dropdown in first time it's appeared in wrong place

**Reply by LeSe0** [reply]
It's bug present in popover too
